lib/stat_tracker.rb

```diff
@@ -20,5 +28,75 @@ def initialize(games, teams, game_teams)
     @games = games
     @teams = teams
     @game_teams = game_teams
+    @game_teams_collection = GameTeamCollection.new("./data/game_teams.csv")
+    @games_collection = GameCollection.new("./data/games.csv")
+    @teams_collection = TeamCollection.new("./data/teams.csv")
+    @game_stats = GameStats.new(@games_collection)
+    @locations = {
```

Comment:
This could be a in interesting refactor for us. It seems like we have some repetition here for sure.